### ISSUE TITLE
Nightly-v1.0.542: Unhandled exceptions caused by weapon listings

### ISSUE THREAD

**Issue by mterry** [chummer5a/chummer5a#523]
Several weapon lists, including Sniper Rifles and Heavy Pistols, are only showing up as chummer.ListItem and are throwing an unhandled exception when accessed in Nightly-v1.0.542.

See screenshots for proof, and exception dump follows.

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

```
************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at Chummer.Weapon.Create(XmlNode objXmlWeapon, Character objCharacter, TreeNode objNode, ContextMenuStrip cmsWeapon, ContextMenuStrip cmsWeaponAccessory, ContextMenuStrip cmsWeaponMod, Boolean blnCreateChildren) in C:\projects\chummer5a\Chummer\Classes\clsEquipment.cs:line 4554
   at Chummer.frmSelectWeapon.lstWeapon_SelectedIndexChanged(Object sender, EventArgs e) in C:\projects\chummer5a\Chummer\frmSelectWeapon.cs:line 130
   at System.Windows.Forms.ListBox.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ListBox.WmReflectCommand(Message& m)
   at System.Windows.Forms.ListBox.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.127.1 built by: NETFXREL3STAGE
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
Chummer5
    Assembly Version: 0.5.177.0
    Win32 Version: 0.5.177.0
    CodeBase: file:///C:/Users/Matthew/Apps/Chummer/Chummer5.exe
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.93.0 built by: NETFXREL2STAGE
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Windows.Forms.DataVisualization
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms.DataVisualization/v4.0_4.0.0.0__31bf3856ad364e35/System.Windows.Forms.DataVisualization.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

![chummer-bug1](https://cloud.githubusercontent.com/assets/387889/11283334/3ad6f704-8ed2-11e5-8c5d-6738b788b8f3.png)
![chummer-bug2](https://cloud.githubusercontent.com/assets/387889/11283335/3ad71af4-8ed2-11e5-9ffc-dc2c915ff111.png)
![chummer-bug3](https://cloud.githubusercontent.com/assets/387889/11283337/3ad8da06-8ed2-11e5-8347-0de9a0ed6a49.png)
![chummer-bug4](https://cloud.githubusercontent.com/assets/387889/11283339/3add4488-8ed2-11e5-858e-2b4b87d6c9a5.png)
![chummer-bug5](https://cloud.githubusercontent.com/assets/387889/11283338/3ad9ec02-8ed2-11e5-9117-95035e340913.png)
![chummer-bug6](https://cloud.githubusercontent.com/assets/387889/11283336/3ad83fd8-8ed2-11e5-8c23-036e173c43ec.png)
![chummer-bug7](https://cloud.githubusercontent.com/assets/387889/11283342/3ae5ab14-8ed2-11e5-9e93-4a057f8cebd3.png)
![chummer-bug8](https://cloud.githubusercontent.com/assets/387889/11283340/3ae2f1a8-8ed2-11e5-81ab-be98cda662a4.png)
![chummer-bug9](https://cloud.githubusercontent.com/assets/387889/11283341/3ae44a80-8ed2-11e5-946f-fa7f3bc56d67.png)
![chummer-bug10](https://cloud.githubusercontent.com/assets/387889/11283343/3ae67ba2-8ed2-11e5-8b2d-8a30f4e6f324.png)
![chummer-bug11](https://cloud.githubusercontent.com/assets/387889/11283344/3ae783bc-8ed2-11e5-9dd2-27d59e0272b6.png)
![chummer-bug12](https://cloud.githubusercontent.com/assets/387889/11283345/3ae853d2-8ed2-11e5-9502-ff7b9e38d3e7.png)
![chummer-bug13](https://cloud.githubusercontent.com/assets/387889/11283346/3aee0886-8ed2-11e5-8246-8d5daa6fe830.png)
![chummer-bug14](https://cloud.githubusercontent.com/assets/387889/11283347/3aeea692-8ed2-11e5-917c-1d557b642b58.png)

**Comment by mterry** [comment]
Still an issue in Nightly-v1.0.544.

**Comment by chummer5a** [comment]
Fixed in 5.178.
